src/components/Editor.svelte

```diff
@@ -56,7 +56,10 @@
       automaticLayout: true,
       readOnly: true,
       folding: true,
-      quickSuggestions: false
+      quickSuggestions: false,
+      wordWrap: "on",
+      unicodeHighlight: {invisibleCharacters: false,
```

Comment:
This object requires formatting

Reply:
Sorry, I'm not familiar with code style of that thing. I should have tried finding a formatter, of course. :(